tensorflow/compiler/xla/service/gpu/convolution_thunk.h

```diff
@@ -115,7 +115,7 @@ class ConvolutionThunk : public Thunk {
       perftools::gputools::dnn::ProfileResult* profile_result);
 
   // Returns the convolve algorithms that can be used for this ConvolutionThunk.
-  std::vector<perftools::gputools::dnn::AlgorithmType> GetAlgorithms(
+  std::vector<perftools::gputools::dnn::AlgorithmDesc::Index> GetAlgorithms(
```

Comment:
Add a TODO to return AlgorithmDesc. The GetAlgorithms itself can go through the loop.

Reply:
Added TODO. Hopefully this completes the PR.